test/info.py

```diff
@@ -105,6 +106,7 @@
     #lpc11u35_arm_watch_efm32_if
     #lpc11u35_arm_watch_nrf51_if
     'Seeed-Arch-Link': 0x9013,
+    'BlueNinja': 0x7010,
```

Comment:
This will need to be updated to match the format here:
https://github.com/mbedmicro/DAPLink/blob/master/test/info.py#L59

Something like:
(   0x7010,     'lpc11u35_archpro_if',              None,               'BlueNinja'            ),